src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Main {
+    public static final int FIGURE_ARRAY_SIZE = 6;
+    public static final int RANDOM_FIGURE_COUNT = 3;

```

Comment:
The constant 'RANDOM_FIGURE_COUNT' is well-named as it describes its purpose. However, ensure that this constant is used according to its name and the task requirements. If the task specifies a different way to determine the count of random figures, you should follow the task instructions.